sap-service-operator/src/main/java/com/sap/cloud/environment/servicebinding/LayeredDataParsingStrategy.java

```diff
@@ -16,16 +24,10 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
-import javax.annotation.Nonnull;
-
-import org.json.JSONException;
-import org.json.JSONObject;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import com.sap.cloud.environment.servicebinding.api.DefaultServiceBinding;
-import com.sap.cloud.environment.servicebinding.api.ServiceBinding;
-
+/**
+ * A {@link LayeredParsingStrategy} that expects all property files to contain "plain" text (i.e. no JSON structures).
```

Comment:
Maybe add an `@see` to the interface. The second sentence doesn't really make sense to me, I mean all properties are recognised by their name. In what format should credentials and metadata be? are they directories instead?

Reply:
> Maybe add an @see to the interface.

Where would you point to?

> I mean all properties are recognised by their name.

Not really. The `LayeredSecretKeyParsingStrategy`, for example, distinguishes metadata and credentials properties by their structure instead - the **one and only** JSON file contains the credentials whereas all other (plain text) files contain metadata. The name of the credentials file doesn't matter.

> In what format should credentials and metadata be? are they directories instead?

Not sure what mean here... The doc already says _"property files [...] contain "plain" text (i.e. no JSON structures)."_.
I thought that would be pretty clear?